src/AspNetCoreModuleV2/CommonLib/FileOutputManager.h

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#pragma once
+
+
+class FileOutputManager : public IOutputManager
+{
+    #define FILE_FLUSH_TIMEOUT 3000
+public:
+    FileOutputManager(PCWSTR pwzStdOutLogFileName, PCWSTR pwzApplciationpath);
+    ~FileOutputManager();
+
+    virtual bool GetStdOutContent(STRU* struStdOutput) override;
```

Comment:
it seems you want to dump the content as ANSI char arrat, you can pass in STRA instead of STRU to avoid encoding

Reply:
Agreed.